<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_instructions">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter the following parameters then click 'Send':</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="status_box">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_param_title_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton_1">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Command:</string>
   </property>
  </widget>
  <widget class="QLabel" name="subSystemCommandPageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Subsystem:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="subSystemTextBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="commandAddressTextBrowser">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>741</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>500</height>
    </size>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>724</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1000</height>
     </size>
    </property>
    <widget class="QTextEdit" name="input_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_input_title_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>230</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_param_title_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_descrip_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_1">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>180</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_6">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>280</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_7">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>330</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_8">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>380</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_9">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>430</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>330</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>430</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="descrip_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>480</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="paramName_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_10">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>480</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
